requirements/doc.txt

```diff
@@ -140,7 +148,7 @@ jsonfield2==4.0.0.post0
     # via
     #   -r requirements/test.txt
     #   edx-celeryutils
-keyring==21.8.0
+keyring==22.0.1
```

Comment:
this is pretty far in a filesystem-oriented dependency, which questionably would affect us. No good changelog, but diff here: https://github.com/jaraco/keyring/compare/v21.8.0...v22.0.0